server/controllers/userController.js

```diff
@@ -63,13 +95,24 @@ class UserController {
    */
   static async verifyUser(req, res) {
     const { email } = req.params;
-    const { status } = req.body;
-    const verifiedUser = await User.verify(email, status, users);
-    return res.status(200).json({
-      status: 200,
-      message: 'User Verification Successful!',
-      data: verifiedUser,
-    });
+    const result = await db.query(verifyUser, [email]);
```

Comment:
why are you making 2 database calls.?

Reply:
The first call updates the user's status, while the other one gets the updated user